### PR TITLE
Add '--minimal' option for app generation

### PR DESCRIPTION
**Pull request by OpakAlex**
### Description of the Change

Add `--minimal` option for app generation. This will disable npm dependencies.

https://github.com/amberframework/amber/issues/1205

### Benefits

Easy to create new API apps

### Possible Drawbacks

Not sure

### CODE REVIEW

File: src/amber/cli/generators/app.cr [hunk 1]
```diff
@@ -14,18 +14,27 @@ module Amber::CLI
...
     def filter(entries)
-      entries.reject { |entry| entry.path.includes?("src/views") && !entry.path.includes?("#{@language}") }
+      entries = entries.reject { |entry| entry.path.includes?("src/views") && !entry.path.includes?("#{@language}") }
+      filter_for_minimal(entries)
+    end
+
+    def filter_for_minimal(entries)
+      return entries unless @minimal
+      filtered = entries.reject { |entry| entry.path.includes?(".js") || entry.path.includes?(".scss") || entry.path.includes?("package.json") || entry.path.includes?("home_controller") }
+      filtered.reject { |entry| entry.path.includes?("src/views") && !entry.path.includes?("mailer") }
```
Comment (drujensen) [inline src/amber/cli/generators/app.cr:37]: reject `config/webpack` as well

Comment (OpakAlex) [reply src/amber/cli/generators/app.cr:38]: fixed

**Comment by OpakAlex** [comment]
@drujensen can you recheck pls?

**Comment by OpakAlex** [comment]
Thanks. How is merging happens here? You have some releases and merge depends on it?

**Pull request merged**